region.go

```diff
@@ -44,115 +44,149 @@ func GetRegion(zone string) (region string, err error) {
 
 // Region describes respective IBM Cloud COS region, VPC region and Zones associated with a region in Power VS.
 type Region struct {
-	Description string
-	VPCRegion   string
-	COSRegion   string
-	Zones       []string
-	SysTypes    []string
+	Description  string
+	VPCRegion    string
+	COSRegion    string
+	Zones        []string
+	SysTypes     []string
+	IPISupported bool
 }
 
 // Regions provides a mapping between Power VS and IBM Cloud VPC and IBM COS regions.
-var Regions = map[string]Region{
+var Regions = FullRegions
+
+// Switch the list of regions to all of the regions.
+func UseFullRegions () {
+	Regions = FullRegions
+}
+
+// Switch the list of regions to only the regions supported by the IPI installer.
+func UseIPIRegions () {
```

Comment:
```suggestion
func GetIPIRegions () map[string]Region {
```
This can be used to filter the IPI regions alone and return it.